configure

```diff
@@ -3912,7 +3966,6 @@ case $_backend in
 		;;
 esac
 
-
 append_var MODULES "backends/platform/$_backend"
```

Comment:
Will this lead to duplication? You already added `"backends/platform/sdl"`

Reply:
backend=kolibrios, so no